DocumentsFromSnapshotMigration/src/test/resources/log4j2.properties

```diff
@@ -33,4 +33,14 @@ rootLogger.appenderRef.console.ref = Console
 
 logger.wireLogger.name = org.apache.http.wire
 logger.wireLogger.level = OFF
-logger.wireLogger.additivity = false
\ No newline at end of file
+logger.wireLogger.additivity = false
+
+# Run and discard TRACE logs from owned packages to test executed paths
+appender.null.type = Null
```

Comment:
Can you name this something a bit more than just 'null'.  It could be confusing to see 'null' if it pops up in the log4j logs if anything is ever off.

Reply:
Updated